lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -676,6 +676,15 @@ def convert_date(date)
       end
 
       def convert_date_no_day(date)
+        year = date[0, 4]
+        month = date[5, 2]
+        day = date[8, 2]
+
+        # somehow, partial dates with the 'XX' (i.e. "2020-01-XX or 2020-XX-XX") are getting past FE validation
+        # fix here in the backend while a proper FE solution is found
+        return year if month == 'XX'
+        return "#{year}-#{month}" if day == 'XX'
+
```

Comment:
Not sure if you've seen any instances of it, but Ive seen like `2019-03` before I think, like to mean sometime in march of 2019. 

```
irb#1(main):031> convert_date_no_day("2019-02")
(irb#1):11:in `parse': invalid date (Date::Error)
	from (irb#1):11:in `convert_date_no_day'
	from (irb#1):31:in `<top (required)>'
```

Not sure if it is worth adding in to account for that. if we arent seeing that, probably doesnt matter... I just thought I recalled seeing that, in the past, and it is not much more effort to support.

Reply:
Haven't seen that yet, but it was easy to support (as you said) so I added that check.